tests/fixtures/pokemon_edits.json

```diff
@@ -0,0 +1,55 @@
+{
+  "compareFunction": "pokemon_edits",
+  "fixtures": [
+  {
+    "description": "pokemon edits",
+        "newVersion": {
+            "type": "Feature",
+            "id": "node!4412655090!2",
+            "properties": {
+                "tourism": "hotel",
+                "name:": "poké",
+                "osm:type": "node",
+                "osm:id": 4412655090,
+                "osm:version": 2,
+                "osm:changeset": 43885598,
+                "osm:timestamp": 1479870698000,
+                "osm:uid": 1306,
+                "osm:user": "PlaneMad"
+            },
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    72.5451861,
+                    23.2005836
+                ]
+            }
+        },
+        "oldVersion": {
+            "type": "Feature",
+            "id": "node!4412655090!1",
+            "properties": {
+                "tourism": "hotel",
+                "name": "Washington Park",
+                "osm:type": "node",
+                "osm:id": 4412655090,
+                "osm:version": 1,
+                "osm:changeset": 42338212,
+                "osm:timestamp": 1474519696000,
+                "osm:uid": 4112063,
+                "osm:user": "kailashdhirwani"
+            },
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    72.5451861,
+                    23.2005836
+                ]
+            }
+        },
+    "expectedResult" : {
+      "result:pokemonEdits": true
+    }
+  }
+  ]
+}
```

Comment:
@amishas157 @krishnanammala - can you please add a negative test as well? i.e. something with a name like "Taj Mahal" or something and test that that `expectedResult` is `{}` or so?